components/FinePrint/FinePrint.js

```diff
@@ -0,0 +1,32 @@
+import styles from './FinePrint.module.css';
+
+export function FinePrint() {
+  return (
+    <div>
+      <ul className={styles.containerList}>
+        <li className={styles.list}>About</li>
+        <li className={styles.list2}>.</li>
```

Comment:
This is what `::after` is exactly for. Check IG's CSS and see how they do it.

Reply:
On it.